modules/ossm-observability-cli.adoc

```diff
@@ -6,7 +6,7 @@
 [id="ossm-observability-cli_{context}"]
 = Accessing metrics and tracing data from the CLI
```

Comment:
I think this section is missing as well

Reply:
Actually this one should be deleted, I split it out into new topis.